code/datums/wounds/cranial_fissure.dm

```diff
@@ -52,12 +52,10 @@
 
 /datum/wound/cranial_fissure/remove_wound(ignore_limb, replaced)
 	REMOVE_TRAIT(limb, TRAIT_IMMUNE_TO_CRANIAL_FISSURE, type)
-	REMOVE_TRAIT(victim, TRAIT_HAS_CRANIAL_FISSURE, type)
-
-	victim.remove_filter(CRANIAL_FISSURE_FILTER_DISPLACEMENT)
-
-	UnregisterSignal(victim, COMSIG_MOB_SLIPPED)
-
+	if (!QDELETED(victim))
```

Comment:
!QDELETED or !victim

Reply:
This can get called during victim's cleanup in which case I don't think we should be bothering with this either?

Reply:
desyncing the trait is non ideal, unreg is not an extra cost either